packages/thirdweb/src/react/web/ui/ConnectWallet/Details.tsx

```diff
@@ -1082,6 +1128,37 @@ export type UseWalletDetailsModalOptions = {
    * ```
    */
   supportedTokens?: SupportedTokens;
+  /**
+   * Customize the NFTs shown in the "View Funds" screen in Details Modal for various networks.
+   *
+   * By default, The "View Funds" screen shows a few popular tokens for default chains and the native token. For other chains it only shows the native token.
+   * @example
+   *
+   * supportedTokens prop allows you to customize this list as shown below which shows "Pudgy Penguins" help when users wallet is connected to Ethereum mainnet.
+   *
+   * ```tsx
+   * import { ConnectButton } from 'thirdweb/react';
+   *
+   * function Example() {
+   *   return (
+   * 		<ConnectButton
+   * 			supportedNFTs={{
+   *        // when connected to Ethereum mainnet - show Pudgy Penguins
+   * 				1: [
+   * 					{
+   * 						address: '0xBd3531dA5CF5857e7CfAA92426877b022e612cf8', // token contract address
+   * 						name: 'Pudgy Penguins',
+   * 						symbol: 'PPG',
+   * 						icon: 'https://i.seadn.io/gcs/files/17ce510a2bd8a559e74fed8a45022ac9.png',
```

Comment:
name/symbol/icon is not accepted here right?

Reply:
Yeah this changed, good catch